.github/workflows/ci.yml

```
@@ -0,0 +1,32 @@
+      - main
+
+jobs:
+  lint-and-build:
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: checkout repostiry
+        uses: actions/checkout@v4
+
+      - name: setup nodejs
+        uses: actions/setup-node@v3
```

actionlint (1.7.7)

the runner of "actions/setup-node@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)